app/compose/Composer.scala

```diff
@@ -7,19 +7,46 @@ class Composer {
               artistInfoResult: Seq[ArtistInfos],
               genresResult: Seq[Genre],
               ratingAverageInfos: Seq[RatingAverageInfo]): MoviesDetail = {
-    val nonEmptyArtistInfoResult = artistInfoResult.filter(res => res.data.nonEmpty)
+    val nonEmptyArtistInfoResult =
+      artistInfoResult.filter(res => res.data.nonEmpty)
     val artistInfosFlatten = nonEmptyArtistInfoResult.flatMap(_.data).distinct
-    val metadata = nonEmptyArtistInfoResult.headOption.map(_.metadata).getOrElse(Metadata(0, 5, 100))
+    val metadata = nonEmptyArtistInfoResult.headOption
+      .map(_.metadata)
+      .getOrElse(Metadata(0, 5, 100))
     val artistInfos = ArtistInfos(metadata, artistInfosFlatten)
     val movieDetails = moviesInfoResult.data.map { movieInfoResult =>
-      val maybeRatingAverageInfo = ratingAverageInfos.find(ratingAvgInfo => ratingAvgInfo.rating.id == movieInfoResult.id)
-      val avgRating: Float = maybeRatingAverageInfo.map(_.rating.rating).getOrElse(-1)
-      val ratedBellowMedian = maybeRatingAverageInfo.forall(_.isBellowMedianAverage)
+      val maybeRatingAverageInfo = ratingAverageInfos.find(ratingAvgInfo =>
+        ratingAvgInfo.rating.id == movieInfoResult.id)
+      val avgRating: Float =
+        maybeRatingAverageInfo.map(_.rating.rating).getOrElse(-1)
+      val ratedBellowMedian =
+        maybeRatingAverageInfo.forall(_.isBellowMedianAverage)
 
-      val genres = movieInfoResult.genres.flatMap { id => genresResult.collect { case genre if genre.id == id => genre.name } }
-      val casts = movieInfoResult.cast.flatMap { id => artistInfos.data.collect { case artistInfo if artistInfo.id == id => Cast(artistInfo.id, artistInfo.gender, artistInfo.name, artistInfo.profilePath)} }
-      MovieDetail(movieInfoResult.id, movieInfoResult.title, averageRating = avgRating, ratedBelowMedian = ratedBellowMedian, movieInfoResult.runtime, movieInfoResult.releaseDate, movieInfoResult.revenue, movieInfoResult.posterPath,
-        movieInfoResult.originalLanguage, genres, casts)
+      val genres = movieInfoResult.genres.flatMap { id =>
+        genresResult.collect { case genre if genre.id == id => genre.name }
+      }
+      val casts = movieInfoResult.cast.flatMap { id =>
+        artistInfos.data.collect {
+          case artistInfo if artistInfo.id == id =>
+            Cast(artistInfo.id,
+                 artistInfo.gender,
+                 artistInfo.name,
+                 artistInfo.profilePath)
+        }
+      }
+      MovieDetail(
```

Comment:
`space` need from val `cast` and new variable

Reply:
reformatted using scalafmt class